tests/kernel/install_ltp.pm

```diff
@@ -25,16 +25,16 @@ sub add_repos {
     zypper_call('--gpg-auto-import-keys ref', dumb_term => 1);
 }
 
+# SLE as registered product
 sub scc_we_enabled {
     script_run("zypper -n products -i", 0);
     return wait_serial(qr/Workstation Extension/);
 }
 
 sub add_we_repo_if_available {
-    # opensuse (doesn't have extensions) or SLE as registered product
-    if (check_var('DISTRI', 'opensuse') || scc_we_enabled) {
-        return;
-    }
+    # opensuse doesn't have extensions
+    return if check_var('DISTRI', 'opensuse');
+
     my $ar_url;
     my $we_repo = get_var('REPO_SLE_WE15_POOL');
     if ($we_repo and check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
```

Comment:
Please replace the `and`s with `&&` while you are at it :-p

Reply:
@richiejp replaced `and` with `&&`. Please merge.